app/code/Magento/Catalog/Model/Product/Attribute/Frontend/Image.php

```diff
@@ -43,8 +48,8 @@ public function getUrl($product)
         $url = false;
         if (!empty($image)) {
             $url = $this->_storeManager->getStore($product->getStore())
-                ->getBaseUrl(\Magento\Framework\UrlInterface::URL_TYPE_MEDIA)
-                . 'catalog/product/' . $image;
+                    ->getBaseUrl(UrlInterface::URL_TYPE_MEDIA)
+                . 'catalog/product' . $image;
```

Comment:
As `$image` may come from various sources there is no guarantee it contains leading slash.

Please 

1. change logic to `. 'catalog/product/' . ltrim($image, '/')` instead 
1. create a separate test case to assure there is no slash duplication
1. as always, do an amend commit and force push

Reply:
ok, thank you

Reply:
@igortregub changes look good now, please squash changes into single commit and force push.

Reply:
@orlangur, done